charts/rancher-gatekeeper-crd/102.0.0+up3.10.0/templates/jobs.yaml

```diff
@@ -40,6 +40,13 @@ spec:
             - name: crd-manifest
               readOnly: true
               mountPath: /etc/config
+          securityContext:
+            seccompProfile:
+                type: RuntimeDefault
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+              - ALL
```

Comment:
For these values outside of the seccomp profile stuff, should we just allow users to provide it as a general `.Values.securityContext` or similar field?

Reply:
done